blogs/whats_new_1.5/index.md

```diff
@@ -0,0 +1,178 @@
+3. Console plugin configuration
+4. Namespace
+5. Deployment model
+6. Agent configuration
+7. Processor configuration
+
+The first significant change is in "Loki client settings".  When you click and open this, you get the following:
+
+![Loki client settings](images/loki_client_settings.png)
+_<div style="text-align: center">Figure 2: Loki client settings</div>_
+
+One of the new fields is "Mode" where you select how you installed Loki.  The most common is "LokiStack", which means you installed the Loki Operator and created a LokiStack instance.  Under the "Loki stack" section, make sure the **Name** matches the LokiStack name you gave it.  The nice part is that it will go and figure out the LokiStack gateway URL for you and give it proper authorization.
+
+Parameters are now exposed under "Console plugin configuration", particularly "Quick filters" (Figure 3).  Network Observability predefines some filters as defaults which can be changed.  While this was possible before, now you can do it in the UI.
+
+![Console plugin configuration](images/console_plugin_configuration.png)
+_<div style="text-align: center">Figure 3: Console plugin configuration</div>_
+
+Under "Agent configuration", there is no longer an agent type because the only supported agent is *eBPF*.  It is still possible to configure *IPFIX* through YAML, but support for OpenShift SDN as the Container Network Interface (CNI) which IPFIX requires, is also deprecated in OpenShift 4.15.
+
+In "Processor configuration", the changes are to enable availability zones, cluster ID, and a "Metrics configuration" section to select a list of predefined metrics under the "include list" (Figure 4).
+
+![Processor configuration](images/processor_configuration.png)
+_<div style="text-align: center">Figure 4: Processor configuration</div>_
+
+The full list of predefined metrics is [here](https://github.com/netobserv/network-observability-operator/blob/main/docs/Metrics.md).  When you include a metric, it stores it in Prometheus and is available as a Prometheus metric prefixed with "netobserv_".  For example, if you add the metric *namespace_egress_bytes_total*, then go to **Observe > Metrics** and enter the PromQL `sum(rate(netobserv_namespace_egress_bytes_total[1m]))`.  This should display a single line that is the sum of the average number of egress bytes over one-minute intervals.  Select a refresh time in the upper right dropdown if you want the graph to be updated periodically.
+
+Availability zones and cluster ID will be covered in the traffic flow table section below.
+
+## UI Changes and Features
+
+The new features and enhancements will be covered by going over the changes in the Network Observability UI, starting with the three tabs in **Observe > Network Traffic**, namely *Overview*, *Network Traffic*, and *Topology*.
+
+### Overview tab
+
+Graphs for Flow RTT and DNS Tracking, including support for DNS TCP (previously only DNS UDP), were added.  There are graphs for:
+
+- Top 5
```

Comment:
Is it worth to mention Max / Avg here ?
There is also the "latest" option in the graphs

Reply:
Updated